### ISSUE TITLE
What is VPAT?

### ISSUE THREAD

**Issue by fulv** [plone/ploneorg.core#272]
https://plone.org/plone-vpat.pdf/view - what is this and why is it on the "More" menu? The link given in description is broken

**Comment by polyester** [comment]
VPAT is a self-assessment on accessibility. It's sort-of required for some gov/semi-gov contracting. It could be gone from the "More" menu.

**Comment by fulv** [comment]
The doormat may be a better place for this than the "More" menu.

**Comment by rioksane** [comment]
Vpat-link can be found in plone.org/accessibility and its not visible in More menu anymore

**Comment by skleinfeldt** [comment]
From @tkimnguyen on Slack:

> Plone-vpat.pdf (https://plone.org/accessibility-1/plone-vpat-1.pdf/view)  is for Plone 5.1, and VPAT2.4RevINT--February2020-Plone.pdf (on https://plone.org/accessibility-1/plone-vpat.pdf/view) is for Plone 5.2

I renamed the File objects to reflect this information, and made links to both of them (making clear which is which) on the [Accessibility page](https://plone.org/accessibility). I also removed the bad link from the description.

So this is really, truly, done!
